openstates/la/legislators.py

```diff
@@ -109,21 +111,24 @@ def scrape_lower_legislator(self, url, leg_info, term):
         cty = xpath_one(infoblk, "./b[contains(text(), 'ASSIGNMENTS')]")
         cty = cty.getnext()
 
-        partyblk = filter(lambda x: "District" in x,
-                          page.xpath('//p[@align="center"]//text()'))[0]
+        #partyblk = filter(lambda x: "District" in x,
+        #                  page.xpath('//td[@width="10%"]//text()'))[0]
 
-        party_flags = {
+        partyblk = []
+	party_flags = {
             "Democrat": "Democratic",
             "Republican": "Republican",
             "Independent": "Independent"
         }
 
-        if leg_info['name'].startswith("Vacant"):
+        if leg_info['name'].startswith(("Vacant","Miguez")):
             return
 
-        party = 'other'
+        #party = 'other'
+	party = 'Republican'
```

Comment:
This is the page I am referring to for lower chamber party data scraping:
http://house.louisiana.gov/H_Reps/H_Reps_FullInfo.asp

As you can see the web team at the Louisiana legislature have neglected to offer the party affiliation info for House members.

Another, note is that this is also true of Upper chamber.